src/python/AsyncStageOut/PublisherWorker.py

```diff
@@ -835,9 +836,23 @@ def format_file_3(self, file):
               'file_parent_list': [{'file_parent_lfn': i} for i in set(file['parents'])],
              }
         file_lumi_list = []
-        for run, lumis in file['runlumi'].items():
-            for lumi in lumis:
-                file_lumi_list.append({'lumi_section_num': int(lumi), 'run_num': int(run)})
+
+        # check if block contains mixed evt/noevt
+        if self.evtLumi:
+            for _, lumis in file['runlumi'].iteritems():
+                for _, evt in lumis.iteritems():
+                   if evt == 'None':
+                       self.evtLumi = False
+                       break
+	    self.logger.debug("Check on the presence of evts/lumi: %s" % self.evtLumi)
```

Comment:
this will always be printed since it is inside the if. Should it be put outside it?